package/MDAnalysis/transformations/nojump.py

```diff
@@ -123,6 +123,11 @@ def _transform(self, ts):
         except np.linalg.LinAlgError:
             msg = f"Periodic box dimensions are not invertible at step {ts.frame}"
             raise NoDataError(msg)
+        if ts.frame == 0:
```

Comment:
Would it be worth adding a small entry in the docs above to detail this case? Maybe just some inline comment of why this is needed would work - I will admit I wasn't able to quickly read through the issues and follow exactly why this was happening.

Reply:
yeah good point, it's not immediately obvious. I've added a comment to try to explain what's going on, let me know if it's not clear

Reply:
Thanks!